Userland/BuggieBox/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+serenity_component(
+    BuggieBox
+    REQUIRED
+    TARGETS BuggieBox
+)
+
+set(SOURCES
+    main.cpp
+    ../Shell/AST.cpp
```

Comment:
These shell files are already a part of libshell, in shell CMake lists. Can we re-use the logic in there that's used for Lagom shell compilation instead of duplicating it here?